app/services/db/document_capture_session/read_verify_doc_status.rb

```diff
@@ -0,0 +1,11 @@
+module Db
+  module DocumentCaptureSession
+    class ReadVerifyDocStatus
+      def self.call(uuid)
+        doc_capture_session = ::DocumentCaptureSession.find_by(uuid: uuid)
+        return unless doc_capture_session && doc_capture_session.verify_doc_submitted_at.present?
+        doc_capture_session.verify_doc_status || :in_progress
```

Comment:
1. this DB column returns a string, but the default value is a symbol, should we make them both strings or both symbols?

2. We have the `ProofingDocumentCaptureSessionResult` from `DocumentCaptureSession` already, can we re-use that instead of having to come up with a new status?

Reply:
nice catch on the symbol... i should probably add a test for that

Reply:
Interesting on point 2.  The story was written up to just receive the status.  I assumed they'd just pull the results from the db afterwards but the client would have no way to do that.  So you are right I should wrap the result in `ProofingDocumentCaptureSessionResult`.

Reply:
Along the lines of #2, the other uses of `DocumentCaptureSession` don't store status or results in Postgres. They infer the status based on the results being stored in Redis and feel like we should be consistent?

Reply:
Yes.  Hooper mentioned he wished he put `DocumentCaptureSession` in redis.

Reply:
This thread was continued on slack...